<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>171</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>程序类型选择</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>程序类别</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="category">
         <item>
          <property name="text">
           <string>请选择程序类别</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>信息输入</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>信息提示</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>子类别</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="subCategory">
         <item>
          <property name="text">
           <string>请选择子类别</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>401</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>流程树</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="addStage">
         <property name="text">
          <string>添加阶段</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteItem">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string>阶段</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>天天</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>十七岁</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>你还差</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>国外</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <item>
           <property name="text">
            <string>新建子项目</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </item>
         </item>
         <item>
          <property name="text">
           <string>新建子项目</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </item>
        </item>
        <item>
         <property name="text">
          <string>新建项目</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>新建子项目</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>发大水</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>飞洒发</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>发生发</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>参数配置说明</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="paramConfig">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="inputPage">
        <widget class="QWidget" name="formLayoutWidget">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>70</y>
           <width>160</width>
           <height>150</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>步骤名称</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="inputProcedureName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>操作者姓名</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="inputOperatorName"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="promptPage">
        <widget class="QWidget" name="formLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>160</width>
           <height>150</height>
          </rect>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>步骤名称</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="promptProcedureName"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>提示信息</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="promptMessage"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="empty"/>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuadd">
    <property name="title">
     <string>操作</string>
    </property>
   </widget>
   <addaction name="menuadd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
